pyproject.toml

```diff
@@ -21,6 +21,7 @@ packages = [
 [tool.poetry.dependencies]
 python = "^3.6.1"
 nautobot = "*"
+nautobot-capacity-metrics = "*"
```

Comment:
Can this be an optional/extras dependency? Or does it need to be a required dependency even if metrics aren't in use?